PhysicsTools/PatAlgos/python/producersLayer1/jetProducer_cfi.py

```diff
@@ -47,7 +47,12 @@
         cms.InputTag("pfCombinedInclusiveSecondaryVertexV2BJetTags"),
         cms.InputTag("softPFMuonBJetTags"),
         cms.InputTag("softPFElectronBJetTags"),
-        cms.InputTag("pfCombinedMVABJetTags")
+        cms.InputTag("pfCombinedMVABJetTags"),
+        cms.InputTag("pfCombinedSecondaryVertexSoftLeptonBJetTags"),
+        cms.InputTag("pfCombinedMVABJetTags"),
+        #CTagging
+        cms.InputTag('pfCombinedCvsLJetTags'),
+        cms.InputTag('pfCombinedCvsBJetTags')
```

Comment:
To prevent PAT tests from failing, I suggest for now to keep the c-tagging additions in this cfi file commented out. Once RelVals with required inputs become available, this can be revisited.